src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/util/ReviewablePreRelease.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.util
+
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.Licence.Companion.SYSTEM_USER
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.Staff
+import java.time.LocalDateTime
+
+interface ReviewablePreRelease {
+  var dateLastUpdated: LocalDateTime?
+  var updatedBy: Staff?
+  var updatedByUsername: String?
+  var reviewDate: LocalDateTime?
+
+  fun markAsReviewed(staff: Staff?) {
+    reviewDate = LocalDateTime.now()
+    dateLastUpdated = LocalDateTime.now()
+    updatedByUsername = staff?.username ?: SYSTEM_USER
+    updatedBy = staff ?: this.updatedBy
+  }
+}
```

Comment:
thought: We have has problems with JPA and lazy loading with entities (Staff) in interfaces that are implemented by entities (TimeServedLicence). Hibernate proxies can have problems with this seems to have problems wrapping them in proxies and then loading the data later on. So make sure you have integration tests that cover all db actions and test manually.